serve.py

```diff
@@ -1,34 +1,110 @@
 import asyncio
-
+import os
 from sanic import Sanic
-from sanic.log import logger, error_logger, access_logger
+from sanic.log import logger
 from sanic import response
-from sanic_cors import CORS, cross_origin
+from sanic_cors import CORS
 import requests
-from sanic.exceptions import abort
 
-app = Sanic("redirector")
+production = True and 'DEV628cc89a6444' not in os.environ
+sem = None
+basedir = os.environ["HOME"] if production else os.getcwd()
+logdir = os.path.join(basedir, "redirector")
+if not os.path.exists(logdir):
+    os.makedirs(logdir, exist_ok=True)
+LOG_SETTINGS = dict(
+    version=1,
+    disable_existing_loggers=False,
+    loggers={
+        "sanic.root": {"level": "INFO", "handlers": ["consolefile"]},
+        "sanic.error": {
+            "level": "INFO",
+            "handlers": ["error_consolefile"],
+            "propagate": True,
+            "qualname": "sanic.error",
+        },
+        "sanic.access": {
+            "level": "INFO",
+            "handlers": ["access_consolefile"],
+            "propagate": True,
+            "qualname": "sanic.access",
+        },
```

Comment:
although "explicit is better than implicit", here I kept looking for differences between specs. If the same, and just different names/targets -- dict comprehension could be used to make it concise and to the point